src/Forms/CustomValidation/Program.cs

```diff
@@ -1,16 +1,19 @@
-﻿using Microsoft.AspNetCore.Blazor.Hosting;
+﻿using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
 
 namespace CustomValidation
 {
-	public class Program
+	public static class Program
 	{
-		public static void Main(string[] args)
+		public async static Task Main(string[] args)
 		{
-			CreateHostBuilder(args).Build().Run();
-		}
+			var builder = WebAssemblyHostBuilder.CreateDefault(args);
+			builder.RootComponents.Add<App>("app");
```

Comment:
When you deleted Startup.cs you forget to move the custom registration across to here. You missed

`builder.Services.AddScoped<PersonValidator>();`

Reply:
done